client/app/components/toolbox_actions.coffee

```diff
@@ -56,7 +54,7 @@ module.exports = ToolboxActions = React.createClass
                 if @props.message?
                     li role: 'presentation',
                         a
-                            href: "raw/#{@props.mailboxID}/#{messageRawId}"
+                            href: "raw/#{@props.message.get('id')}"
```

Comment:
This is not related to cozydb.

It is a seprate bugfix :  in conversation you could not see some raw messages.
Exemple : you have message A in inbox and message B in sent. You display the conversation from message A and it shows message B, but the message B is not in inbox so the messageRawID was undefined.